### PR TITLE
fixing geometry manager test to be more portable

### PR DESCRIPTION
**Pull request by sslattery**
Fixing the geometry manager test failing on the CI to be more portable. Random numbers were heavily used in this test. Replacing with a deterministic parallel test.

### CODE REVIEW

File: packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp [hunk 1]
```diff
@@ -61,11 +61,11 @@ TEUCHOS_UNIT_TEST( GeometryManager, geometry_manager_cylinder_test )
...
-	double length = (double) std::rand() / RAND_MAX;
-	double radius = (double) std::rand() / RAND_MAX;
-	double centroid_x = (double) std::rand() / RAND_MAX - 0.5 + my_rank;
-	double centroid_y = (double) std::rand() / RAND_MAX - 0.5 + my_rank;
-	double centroid_z = (double) std::rand() / RAND_MAX - 0.5 + my_rank;
+	double length = 1.0;
+	double radius = 1.0;
+	double centroid_x = i+my_rank;
+	double centroid_y = i+my_rank;
+	double centroid_z = i+my_rank;
```
Comment (dalg24) [inline packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp]: what was wrong about using random numbers?

File: packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp [hunk 2]
```diff
@@ -76,69 +76,29 @@ TEUCHOS_UNIT_TEST( GeometryManager, geometry_manager_cylinder_test )
...
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[2] , z_min-my_rank, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[3] , x_max+my_size-my_rank-1, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[4] , y_max+my_size-my_rank-1, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[5] , z_max+my_size-my_rank-1, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[0] , -1.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[1] , -1.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[2] , -0.5, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[3] , my_size+99.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[4] , my_size+99.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[5] , my_size+98.5, 1.0e-14 );
```
Comment (dalg24) [inline packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp]: why does ``local_box`` is tested ``TEST_EQUALITY`` while ``global_box`` uses ``TEST_FLOATING_EQUALITY``?

File: packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp (hunk 1 above)
Comment (dalg24) [inline packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp]: you can feed ``std::stand`` a seed if you are worried about reproducibility.

**Comment by sslattery** [comment]
The floating equality test is a left over from the previous test of using random numbers - I guess I could change this although it doesn't affect the test results.

As far as getting rid of the random numbers, I ultimately didn't like the test anyway - it was written several years ago. The new test is easier to understand.

**Pull request merged**